content/en/docs/setup/best-practices/node-conformance.md

```diff
@@ -19,7 +19,7 @@ To run node conformance test, a node must satisfy the same prerequisites as a
 standard Kubernetes node. At a minimum, the node should have the following
 daemons installed:
 
-* Container Runtime (Docker)
+* Container Runtime (containerd or crio)
```

Comment:
there are many container runtimes out there, why are we mentioning only these two?

Reply:
@tengqm I understand there are many container runtime out there but instead of mentioning every container runtime I mentioned a few of them and etc (there can be some other possibilities too) do you want me to mention every container runtime over here?

Reply:
The problem as I see it, is that this revision makes the reader believe that there are only options out there. One is `containerd` and the other is `cri-o`.
We are introducing new defects/confusion while fixing the old one.

Reply:
would you like to suggest as etc which means there can be other container runtimes too? or i can write (CRI-compatible)

Reply:
I would suggest to add some more popular container technologies.
eg:  Docker, cri-o, podman, rkt

Reply:
I agree with tengqm, I am also reading it as only these two are the option.

I think tamilselvan1102 idea is good so we just mention a few as examples. something like below?

* Container Runtime (For example: Docker, cri-o, podman, rkt and a few more)

Reply:
I added a few more container runtime and etc. I think now it looks more clear to a reader to believe there are more container runtime out there